Makefile

```diff
@@ -1,65 +1,29 @@
-UA_SCRIPTS = ubuntu-advantage $(wildcard modules/*)
-MOTD_SCRIPTS = $(wildcard update-motd.d/*)
 
 
 build:
 	@echo Nothing to be done for build
 
-deb-trusty:
-	# TODO provide proper py2 py3 package build script
-	@echo Building unsigned debian package for trusty 14.04.1
-	@which dpkg-buildpackage || \
-               { echo -e "Missing build dependencies. Install with:" \
-                 "\n sudo apt-get install devscripts"; exit 1; }
-	./dev/bddeb --series trusty
-
-ci-deps:
-	@echo Installing Continuous Integration dependencies
+deps:
 	./dev/read-dependencies -v 3 --system-pkg-names --test-distro
 
+clean:
+	rm -f *.build *.buildinfo *.changes .coverage *.deb *.dsc *.tar.gz
+	rm -rf *.egg-info/ .tox/
+	find . -type f -name '*.pyc' -delete
+	find . -type d -name '*__pycache__' -delete
+
 deb:
 	@echo Building unsigned debian package
 	@which dpkg-buildpackage || \
                { echo -e "Missing build dependencies. Install with:" \
                  "\n make ci-deps"; exit 1; }
```

Comment:
This refers to the `ci-deps` target you've removed.

Reply:
good catch thanks!